tests/test_io.py

```diff
@@ -0,0 +1,217 @@
+import configparser
+import logging
+import os
+import shutil
+
+import pytest
+
+import py
+from bidict import bidict
+from edges_io import io
+
+LOAD_ALIASES = bidict(
+    {
+        "ambient": "Ambient",
+        "hot_load": "HotLoad",
+        "open": "LongCableOpen",
+        "short": "LongCableShorted",
+    }
+)
+
+LOGGING = logging.getLogger("edges-io")
+
+
+@pytest.fixture("module")
+def create_testdir(tmp_path_factory):
+    testDir = create_test_env(tmp_path_factory)
+    return testDir
+
+
+@pytest.fixture("module")
+def create_test_env(tmp_path_factory):
+    # Create an ideal observation file using tmp_path_factory
+    pthList = ["Spectra", "Resistance", "S11"]
+    s11List = [
+        "Ambient",
+        "AntSim3",
+        "HotLoad",
+        "LongCableOpen",
+        "LongCableShorted",
+        "ReceiverReading01",
+        "ReceiverReading02",
+        "SwitchingState01",
+        "SwitchingState02",
+    ]
+    a = tmp_path_factory.mktemp("Test_Obs")
+    b = a / "Receiver01_2020_01_01_010_to_200MHz"
+    b.mkdir()
+    d = b / "25C"
+    d.mkdir()
+    note = d / "Notes.txt"
+    note.touch()
+    dlist = []
+    slist = []
+    for i, p in enumerate(pthList):
```

Comment:
While I usually encourage using for loops, in this case, the for loop doesn't achieve anything -- you might as well just do each of the three sub directories individually. It _may_ help to separate each into a function (that would make the fixture read a little easier). So you could have

```python
make_spectra_dir(d)
make_resistance_dir(d)
make_s11_dir(d)
```